app/drivers/sensor/battery/Kconfig

```diff
@@ -1,20 +1,25 @@
 # Copyright (c) 2020-2021 The ZMK Contributors
 # SPDX-License-Identifier: MIT
 
+DT_COMPAT_ZMK_BATTERY_NRF_VDDH := zmk,battery-nrf-vddh
+DT_COMPAT_ZMK_BATTERY_VOLTAGE_DIVIDER := zmk,battery-voltage-divider
+
 config ZMK_BATTERY
 	bool "ZMK battery monitoring"
 	help
 		Enable battery monitoring
 
 config ZMK_BATTERY_NRF_VDDH
-	bool "ZMK nRF VDDH battery monitoring"
+	bool
```

Comment:
I followed the convention I saw in Zephyr of not labeling these (you shouldn't ever have to manually set them anymore) and making a `DT_COMPAT_*` constant to use with `${dt_compat_enabled, ...)`.